frontend/ui/data/css_tweaks.lua

```diff
@@ -1045,6 +956,29 @@ This tweak can be duplicated as a user style tweak when books contain footnotes
         (function()
             local sub_table = {
                 title = _("In-page footnote font size"),
+                {
+                    id = "inpage_footnote_font-size_smaller",
+                    conflicts_with = function(id) return util.stringStartsWith(id, "inpage_footnote_font-size_") end,
+                    title = _("Smaller footnotes (80%)"),
+                    description = _([[
+Decrease size of in-page footnotes.]]),
+                    css = [[
+body[name="notes"] > section {
+    -cr-only-if: fb2-document;
+        font-size: 0.75rem;
+}
+body[name="comments"] > section {
+    -cr-only-if: fb2-document;
+        font-size: 0.85rem;
+}
+*, autoBoxing {
+    -cr-hint: late;
+    -cr-only-if: -fb2-document inpage-footnote;
+        font-size: 0.8rem !important;
+}
+                    ]],
+                    separator = true,
+                },
```

Comment:
OK, what I disliked in the other PR, I'm ok with having it only in this "somehow legacy" tweak.
May be it could have a more verbose description, mentionning it's limited (no inheritance, so possibly no effect - the reasong why you qualitied the others with "force" all along).

Reply:
I'm honestly not 100% sure if I like this better or if it's better to keep the fb2 footnote size tweak separate. My main concern was cleaning up the top-level footnote menu and I got a bit carried away.

As you noted, it prevents some combinations like having larger epub and smaller fb2.
Any preference on your side for reducing menu items vs keeping fb2 size separate? (as in master, or also moving it inside font size)

Reply:
Dunno, not sure I get what the options are :).
I don't mint _not_ reducing menu stuff for FB2 and having them separated (FB2 is useful to user from a few countries only, 98% of users won't ever hear about FB2).
I'd rather not have the font size % duplicated in the FB2 submenu.

Reply:
I reverted the changes to fb2, incorporating that into the migration would have been tricky anyway.

It now just removes the "smaller" top-level variants and moves them to the footnote font-size submenu.